### ISSUE TITLE
BUG: printHotkeyForCommand throw error 

### ISSUE THREAD

**Issue by jerviscui** [nothingislost/obsidian-hover-editor#242]
Starting up with the plugin turned on generates an exception:

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'length')
    at e.printHotkeyForCommand (app.js:1:1096439)
    at t.<anonymous> (app.js:1:1299151)
    at app.js:1:237027
    at Object.next (app.js:1:237132)
    at app.js:1:236048
    at new Promise (<anonymous>)
    at v (app.js:1:235793)
    at t.onOpen (app.js:1:1298948)
    at t.<anonymous> (app.js:1:1291071)
    at app.js:1:237027
```

**Comment by pjeby** [comment]
This is an Obsidian bug; I've been told it was introduced July 25th and will be fixed in the next patch release.  You can also roll back to an Obsidian release dated before July 25.

**Comment by pjeby** [comment]
Oh, and also, as far as I can tell it does not actually affect the operation of Hover Editor, or anything else in Obsidian.  It's a side effect of Hover Editor creating a hidden graph view to patch the graph plugin to support hover editors.  The patch still goes through even with this error, so HE's actual functionality should be unimpaired.

**Comment by jerviscui** [comment]
Okay, thank you very much.

**Comment by GitMurf** [comment]
thanks for the info and confirmation here! I noticed this error recently and wasn't sure where it was coming from :)

**Comment by pjeby** [comment]
It's fixed in the current Obsidian 1.5 alpha preview, so it should be gone before too long.
